src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -402,23 +418,32 @@ def post(self, request: Request, organization) -> Response:
             )
             save_team_assignments(om, teams)
 
-        if settings.SENTRY_ENABLE_INVITES and result.get("sendInvite"):
-            referrer = request.query_params.get("referrer")
-            om.send_invite_email(referrer)
-            member_invited.send_robust(
-                member=om, user=request.user, sender=self, referrer=request.data.get("referrer")
+        if is_active_staff(request):
+            self.create_audit_entry(
+                request=request,
+                organization_id=organization.id,
+                target_object=om.id,
+                data=om.get_audit_log_data(),
+                event=(audit_log.get_event_id("MEMBER_ADD")),
             )
 
-        self.create_audit_entry(
-            request=request,
-            organization_id=organization.id,
-            target_object=om.id,
-            data=om.get_audit_log_data(),
-            event=(
-                audit_log.get_event_id("MEMBER_INVITE")
-                if settings.SENTRY_ENABLE_INVITES
-                else audit_log.get_event_id("MEMBER_ADD")
-            ),
-        )
+        else:
+            if settings.SENTRY_ENABLE_INVITES and result.get("sendInvite"):
+                referrer = request.query_params.get("referrer")
+                om.send_invite_email(referrer)
+                member_invited.send_robust(
+                    member=om, user=request.user, sender=self, referrer=request.data.get("referrer")
+                )
+            self.create_audit_entry(
+                request=request,
+                organization_id=organization.id,
+                target_object=om.id,
+                data=om.get_audit_log_data(),
+                event=(
+                    audit_log.get_event_id("MEMBER_INVITE")
+                    if settings.SENTRY_ENABLE_INVITES
+                    else audit_log.get_event_id("MEMBER_ADD")
+                ),
+            )
```

Comment:
I think you can clean this up to be a little simpler. Just rough code, but something like:
```
if is_active_staff(member):
   event = (audit_log.get_event_id("MEMBER_ADD"))
else:
    event=(
                    audit_log.get_event_id("MEMBER_INVITE")
                    if settings.SENTRY_ENABLE_INVITES
                    else audit_log.get_event_id("MEMBER_ADD")
                )
       if settings.SENTRY_ENABLE_INVITES and result.get("sendInvite"):
                referrer = request.query_params.get("referrer")
                om.send_invite_email(referrer)
                member_invited.send_robust(
                    member=om, user=request.user, sender=self, referrer=request.data.get("referrer")
                )
self.create_audit_entry(
                request=request,
                organization_id=organization.id,
                target_object=om.id,
                data=om.get_audit_log_data(),
                event=event,
            )
```

You could even pull this stuff out to it's own function to keep this area more contained and easier to read/navigate/read